scipy/optimize/tests/test_linprog.py

```diff
@@ -1648,15 +1644,11 @@ def test_bug_6139_low_tol(self):
         # https://github.com/scipy/scipy/issues/6139
         # Without ``presolve`` eliminating such rows the result is incorrect.
         self.options.update({'tol': 1e-12})
-        with pytest.raises(ValueError):
+        with pytest.raises(AssertionError, match='linprog status 4'):
```

Comment:
I probably could have skipped this, too, but I thought I might as well replace one of these five tests. It's not pretty, but it does check that status 4 is returned (instead of the `ValueError`) per the change above.